.travis.yml

```diff
@@ -5,3 +5,6 @@ rvm:
   - 2.0.0
   - 2.1.2
 script: bundle exec rspec
+before_install:
+  - gem update --system 2.4.8
```

Comment:
I gather that this is to work around the incompatibility between the Bundler and RubyGems versions on Travis CI, but we don’t need to update RubyGems itself to fix that, only Bundler. So I don’t think we need to `gem update --system` here at all.